index.bs

```diff
@@ -3249,7 +3256,6 @@ extended attribute’s [=takes a named argument list|named argument list=].
 
 Two types are <dfn id="dfn-distinguishable" export>distinguishable</dfn> if
 at most one of the two [=includes a nullable type=]
-or is a [=dictionary type=],
```

Comment:
The table doesn't need to be satisfied if you meet either one of the following two conditions (I'm not too sure they apply frankly, still reading up on union types specifics).